openownership/data-standard
# Deleting unused docs pages, fixes #433

# Overview

- What does this pull request do?

Deletes a load of old, stub pages and unused docs sections. They were not linked to the main docs pages and navigation and therefore causing build errors (#433).

- How can a reviewer test or examine your changes?

Check the content of the pages deleted to check that the content is old/outdated. Build the docs & see the lack of warning messages.

- Who is best placed to review it?

@siwhitehouse 

(Closes/Relates to) issue: #

## Translations

n/a

## Documentation & Release

n/a
